packages/cli/vite.config.unit.ts

```diff
@@ -10,12 +10,37 @@ export default defineConfig({
     cache: {
       dir: '../../node_modules/.vitest',
     },
+    alias: [
+      {
+        find: '@code-pushup/test-utils',
+        replacement: new URL('../../testing/test-utils/src', import.meta.url)
+          .pathname,
+      },
+      {
+        find: '@code-pushup/core',
+        replacement: new URL('../core/src', import.meta.url).pathname,
+      },
+      {
+        find: '@code-pushup/models',
+        replacement: new URL('../models/src', import.meta.url).pathname,
+      },
+      {
+        find: '@code-pushup/utils',
+        replacement: new URL('../utils/src', import.meta.url).pathname,
+      },
+    ],
+    pool: 'threads',
+    poolOptions: {
+      threads: {
+        singleThread: true,
+      },
+    },
     coverage: {
       reporter: ['lcov'],
     },
     environment: 'node',
     include: ['src/**/*.unit.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-    globalSetup: ['global-setup.ts'],
+    globalSetup: ['../../global-setup.ts'],
```

Comment:
for some reason, the `globalSetup` file paths are now resolved from the project root.